Automattic/wp-calypso
# Gutenberg: Update block inspector margins

Fix some visual glitches in the Gutenberg block inspector.

- Options bottom margin (for the blocks that have options other than Advanced).
- Advanced option bottom margin.
- Block card bottom margin (for the blocks that don't have options except Advanced)

| Before | After |
| - | - |
| <img width="293" alt="screenshot 2018-09-27 at 14 26 19" src="https://user-images.githubusercontent.com/2070010/46149233-c525b880-c261-11e8-9689-ff4add5ed887.png"> | <img width="292" alt="screenshot 2018-09-27 at 14 26 08" src="https://user-images.githubusercontent.com/2070010/46149246-c9ea6c80-c261-11e8-9c47-1ca689aae2fd.png"> |
| <img width="289" alt="screenshot 2018-09-27 at 14 25 45" src="https://user-images.githubusercontent.com/2070010/46149267-d40c6b00-c261-11e8-8152-66cdb719661b.png"> | <img width="289" alt="screenshot 2018-09-27 at 14 25 54" src="https://user-images.githubusercontent.com/2070010/46149275-d8388880-c261-11e8-9713-caaabcceef11.png"> |

## Testing instructions

- Open Gutenberg in Calypso (http://calypso.localhost:3000/gutenberg/post/) and look at the sidebar for:
  - Blocks that have options other than advanced (e.g. Paragraph).
  - Blocks that don't (e.g. Quote).
- Make sure they look like the example screenshots.
